UnityProject/Assets/Scripts/HealthV2/Living/Mutations/BigStomachMutation.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "BigStomachMutation", menuName = "ScriptableObjects/Mutations/BigStomachMutation")]
+public class BigStomachMutation : MutationSO
+{
+
+	public override Mutation GetMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO)
+	{
+		return new InBigStomachMutation(BodyPart,_RelatedMutationSO);
+	}
+
+	public class InBigStomachMutation : Mutation
+	{
+		public InBigStomachMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO) : base(BodyPart,_RelatedMutationSO)
+		{
+
+		}
+
+		public override void SetUp()
+		{
+			var Stomach = BodyPart.GetComponent<Stomach>();
+			Stomach.StomachContents.SetMaxCapacity(99); //idk Custom thing, if it's preset custom
+		}
+
+		public override void Remove()
+		{
+			var Stomach = BodyPart.GetComponent<Stomach>();
+			Stomach.StomachContents.SetMaxCapacity(2);
```

Comment:
another magic number

Reply:
going to think of a better system that resets it to the default stomach size